### PR TITLE
Fixed incorrect sha, actions run on PR

### PR DESCRIPTION
**Pull request by Trey2k**

### CODE REVIEW

File: .github/workflows/linux.yml [hunk 1]
```diff
@@ -38,7 +43,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: godotengine/godot
-          ref: 1a1a450
+          ref: 7e79aea
```
Comment (DeeJayLSP) [inline .github/workflows/linux.yml]: Only the SHA doesn't seem to work. can we try `master/7e79aea`?

**Review by DeeJayLSP — CHANGES_REQUESTED** [review]

File: .github/workflows/linux.yml (hunk 1 above)
Comment (Trey2k) [reply .github/workflows/linux.yml]: Using the full sha (7e79aead99a53ee7cdf383add9a6a2aea4f15beb) seems to work.

**Comment by DeeJayLSP** [comment]
Should be OK to merge after a squash

**Comment by Trey2k** [comment]
> Should be OK to merge after a squash

Merging is blocked without an approving review.

**Review by DeeJayLSP — APPROVED** [review]

**Pull request merged**